WireDomain/Sources/WireDomain/Utilities/Journal/JournalKey.swift

```diff
@@ -69,6 +69,30 @@ public extension JournalKey where Value == Bool {
 
 }
 
+public extension JournalKey where Value == Bool? {
+
+    /// Whether the user has given consent to track analytics.
+
+    static let isAnalyticsTrackingConsentGiven = Self(
+        "isAnalyticsTrackingConsentGiven",
+        defaultValue: nil
+    )
+
+}
+
+public extension JournalKey where Value == String? {
+
+    /// If the user went through the flow of registering a new personal account and gave consent to analytics tracking,
+    /// the newly created analytics id is temporarily stored in this property. After setting up the user session this
+    /// property will be cleared and the value stored in the database under `ZMUser.analyticsIdentifier` property.
+
+    static let analyticsIDFromRegistration = Self(
```

Comment:
thought: I'm wondering if we could return the analytics id assigned during the registration flow back over the bridge like we do with the regular user id. My concern (not sure if it's valid) is that we start to use the Journal as a general storage mechanism (its intention was to support interruptible operations). What do you think?